Main.java

```diff
@@ -10,6 +11,37 @@
    */
   public static void main(String[] args) {
     // TODO code application logic here
-    
+    Scanner input = new Scanner(System.in);
+    System.out.println ("Please enter how much you spent");
+    int money_spent = input.nextInt();
+    if (money_spent <= 40) {
+      System.out.println ("You get 10% off your order!");
+      double money_saved = money_spent * 0.10;
+      System.out.println ("You saved " + money_saved + "$!");
+      double total = money_spent - money_saved;
+      System.out.println ("Your new total is " + total + "$!");
+    }
+    else if (money_spent <= 80) {
+      System.out.println ("You get 20% off your order!");
+      double money_saved = money_spent * 0.2;
+      System.out.println ("You saved " + money_saved + "$!");
+      double total = money_spent - money_saved;
+      System.out.println ("Your new total is " + total + "$!");
+    }
+    else if (money_spent <= 120) {
+      System.out.println ("You get 30% off your order!");
+      double money_saved = money_spent * 0.3;
+      System.out.println ("You saved " + money_saved + "$!");
+      double total = money_spent - money_saved;
+      System.out.println ("Your new total is " + total + "$!");
+    }
```

Comment:
Make sure that you are adding comments to your code. Think of the steps your program makes and turn these into comments. Though, good job at forming the if statements to allow you to simplify the conditions. Good work.